monkey/agent_plugins/exploiters/mssql/src/mssql_options.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from pydantic import Field
+
+from common.base_models import InfectionMonkeyBaseModel
+from common.types import NetworkPort
+
+
+class MSSQLOptions(InfectionMonkeyBaseModel):
+    target_ports: List[NetworkPort] = Field(
+        default=[1433],
+        description="A list of TCP ports that the MSSQL exploiter will try to exploit.",
+    )
+    try_all_discovered_tcp_ports: bool = Field(
+        default=False,
+        description="Attempt to exploit MSSQL on all TCP ports discovered from network scanning.",
+    )
```

Comment:
I don't know that we want to try on all discovered ports. Just ports where we have positively identified that MSSQL is running.

Reply:
`try_all_mssql_open_ports` ?